index.xml

```diff
@@ -8626,6 +8748,11 @@ context.setColorEnforcedRange(-1, 255, 256);</x:codeblock>
               <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>
               <span class='rfc2119'>MUST</span> be a subset of the interface's
               <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>.
+              Similarly, if <a class='xattr' href='#Exposed'>[Exposed]</a> appears on both a
+              namespace and one of its namespace members, then the namespace member's
+              <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>
+              <span class='rfc2119'>MUST</span> be a subset of the namespace's
+              <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>.
```

Comment:
The spec seems to be missing an obvious requirement for interface that we should add, and add a corresponding one for namespaces: "If [Exposed] appears on a partial interface, then the partial interface's exposure set MUST be a subset of the interface's exposure set."  Otherwise we would be allowing this:

    interface Foo {
    };
    [Exposed=Worker]
    partial interface Foo {
      void func();
    };